sdk/hybridcompute/Azure.ResourceManager.HybridCompute/src/Generated/HybridComputeMachineData.cs

```diff
@@ -32,21 +37,135 @@ public HybridComputeMachineData(AzureLocation location) : base(location)
+        /// <summary> Specifies the cloud provider (Azure/AWS/GCP...). </summary>
+        public string CloudMetadataProvider
+        {
+            get => CloudMetadata is null ? default : CloudMetadata.Provider;
+        }
+
+        /// <summary> The info of the machine w.r.t Agent Upgrade. </summary>
+        public AgentUpgrade AgentUpgrade { get; set; }
+        /// <summary> Specifies the operating system settings for the hybrid machine. </summary>
+        public OSProfile OSProfile { get; set; }
+        /// <summary> The provisioning state, which only appears in the response. </summary>
+        public string ProvisioningState { get; }
+        /// <summary> The status of the hybrid machine agent. </summary>
+        public HybridComputeStatusType? Status { get; }
+        /// <summary> The time of the last status change. </summary>
+        public DateTimeOffset? LastStatusChange { get; }
+        /// <summary> Details about the error state. </summary>
+        public IReadOnlyList<ResponseError> ErrorDetails { get; }
+        /// <summary> The hybrid machine agent full version. </summary>
+        public string AgentVersion { get; }
+        /// <summary> Specifies the hybrid machine unique ID. </summary>
+        public string VmId { get; set; }
+        /// <summary> Specifies the hybrid machine display name. </summary>
+        public string DisplayName { get; }
+        /// <summary> Specifies the hybrid machine FQDN. </summary>
+        public string MachineFqdn { get; }
+        /// <summary> Public Key that the client provides to be used during initial resource onboarding. </summary>
+        public string ClientPublicKey { get; set; }
+        /// <summary> The Operating System running on the hybrid machine. </summary>
+        public string OSName { get; }
+        /// <summary> The version of Operating System running on the hybrid machine. </summary>
+        public string OSVersion { get; }
+        /// <summary> The type of Operating System (windows/linux). </summary>
+        public string OSType { get; set; }
+        /// <summary> Specifies the Arc Machine's unique SMBIOS ID. </summary>
+        public string VmUuid { get; }
+        /// <summary> Machine Extensions information (deprecated field). </summary>
+        public IList<MachineExtensionInstanceView> Extensions { get; }
+        /// <summary> Specifies the Operating System product SKU. </summary>
+        public string OSSku { get; }
+        /// <summary> Specifies the Windows domain name. </summary>
+        public string DomainName { get; }
+        /// <summary> Specifies the AD fully qualified display name. </summary>
+        public string AdFqdn { get; }
+        /// <summary> Specifies the DNS fully qualified display name. </summary>
+        public string DnsFqdn { get; }
+        /// <summary> The resource id of the private link scope this machine is assigned to, if any. </summary>
+        public string PrivateLinkScopeResourceId { get; set; }
```

Comment:
What's the format of this property value?
- If it's a GUID, use type [Guid](https://learn.microsoft.com/en-us/dotnet/api/system.guid?view=net-6.0). See example [here](https://apiview.dev/Assemblies/Review/25a98e223bda4f39b4ef76bebffa2c23#d4cebe383870410cab2e14d0709f78bb).
- If it's a resource identifier, use type [ResourceIdentifier](https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/core/Azure.Core/src/ResourceIdentifier.cs). See example [here](https://apiview.dev/Assemblies/Review/25a98e223bda4f39b4ef76bebffa2c23#9f6fc0ae7c8e497e98d7979dadf97a4c).

Reply:
Same question for `ParentClusterResourceId`.

Reply:
@archerzz I got an error opening the example links although being a member of the microsoft org. Wonder if you can send out another link or screenshot for the ResourceIdentifier example.

![image](https://github.com/Azure/azure-sdk-for-net/assets/137821665/2715dbc1-e1f5-420b-801a-caaecd7af199)